plugins/tiddlywiki/tiddlyweb/icon-cloud.tid

```diff
@@ -0,0 +1,11 @@
+title: $:/plugins/tiddlywiki/tiddlyweb/icon/cloud
+tags: $:/tags/Image
+
+<svg class="tc-image-cloud tc-image-button" width="22pt" height="22pt" viewBox="0 0 128 128">
```

Comment:
I think this one can be much lighter: 951 char vs 2562

```
<svg xmlns="http://www.w3.org/2000/svg" class="tc-image-cloud tc-image-button" width="22pt" height="22pt" viewBox="0 0 128 128">
<g stroke="none" fill-rule="nonzero">
  <path d="M24 103a24 24 0 01-9.97-45.84 24 24 0 0128-24.83A36 36 0 01106 55v.1a24 24 0 011 47.72v.19H24z"/>
  <path class="tc-image-cloud-idle" d="M57.93 84.7a6 6 0 01-8.49 0L35.3 70.56a6 6 0 118.49-8.49l9.9 9.9L81.97 43.7a6 6 0 018.49 8.48L57.93 84.7z"/>
  <path class="tc-image-cloud-progress tc-animate-rotate-slow" d="M44.8 40a3.6 3.6 0 100 7.2h2.06A24 24 0 0063.6 88h.4v-7.2a16.8 16.8 0 01-16.8-16.48V64c0-5.17 2.33-9.79 6-12.87v4.47a3.6 3.6 0 107.2 0v-12a3.6 3.6 0 00-3.6-3.6h-12zM64 40v7.2a16.8 16.8 0 0110.8 29.67V72.4a3.6 3.6 0 10-7.2 0v12a3.6 3.6 0 003.6 3.6h12a3.6 3.6 0 100-7.2h-2.06A24 24 0 0064 40z"/>
  <path class="tc-image-cloud-error tc-animate-pulse-fast" d="M64 84a6 6 0 110 12 6 6 0 010-12zm0-52a6 6 0 016 6v34a6 6 0 11-12 0V38a6 6 0 016-6z"/>
</g>
</svg>
```

Reply:
Thanks @sycom and apologies for the delay. I added the compression here: aaa3b34d2